sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml

```diff
@@ -1,16 +1,7 @@
-friendly_name: Braze Users
-description: |-
-  User Data from the CTMS emails table
-
-  See https://mozilla-hub.atlassian.net/browse/DENG-3008
-owners:
-- cbeck@mozilla.com
-labels:
-  incremental: false
-  schedule: daily
-  owner: cbeck
-bigquery:
-  time_partitioning: null
-scheduling:
-  dag_name: bqetl_braze
-  date_partition_parameter: null
+friendly_name : Braze Users description : | -User Data
```

Comment:
you only need the | if you go to the next line no?

Reply:
this accidentally got in there